TasksApp/TasksApp/Application/UI/Screens/Geolocation/GeolocationViewController.swift

```diff
@@ -0,0 +1,209 @@
+//
+//  GeolocationViewController.swift
+//  TasksApp
+//
+//  Created by user on 23.05.2024.
+//
+
+import UIKit
+import CoreLocation
+import SnapKit
+
+final class GeolocationViewController: UIViewController {
+    private let longitudeTitle : UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.font = UIFont.systemFont(ofSize: 22)
+        return label
+    }()
+    
+    private let latitudeTitle : UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.font = UIFont.systemFont(ofSize: 22)
+        return label
+    }()
+    
+    private let timestampTitle : UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.font = UIFont.systemFont(ofSize: 22)
+        return label
+    }()
```

Comment:
can we do something about code repetition?

Reply:
I think, yes.
Move label creation to specific method.